docs-en/chapter_array_and_linkedlist/linked_list.md

```diff
@@ -406,9 +406,9 @@ In contrast, the time complexity of inserting an element in an array is $O(n)$,
 
 ### Deleting a Node
 
-As shown below, deleting a node in a linked list is also very convenient, **requiring only the change of one node's reference (pointer)**.
+As illustrated in Figure 4-7, deleting a node from a linked list is also straightforward, **involving only the modification of a single node's reference (pointer)**.
 
-Note that although node `P` still points to `n1` after the deletion operation is completed, it is no longer accessible when traversing the list, meaning `P` is no longer part of the list.
+It's important to note that even though node `P` continues to point to `n1` after being deleted, it becomes inaccessible during list traversal. This effectively means that `P` is no longer a part of the list.
```

Comment:
"during list traversal" -> "during linked list traversal", "a part of the list" -> "a part of the linked list" (list and linked list are two separate concepts).